regression/contracts/assigns_replace_07/main.c

```diff
@@ -4,22 +4,22 @@
 
 struct pair
```

Comment:
⛏️  That's not really a `pair` anymore.

Reply:
Nice catch! Thanks, will fix it + push a couple more minor changes and then merge.

Reply:
Fixed.